CHANGELOG.md

```diff
@@ -31,10 +45,20 @@ Changes by Version
     ```
 ##### New Features
 
+- Support secure gRPC channel between agent and collector ([#1391](https://github.com/jaegertracing/jaeger/pull/1391), [@ghouscht](https://github.com/ghouscht), [@yurishkuro](https://github.com/yurishkuro))
+
 ##### Bug fixes, Minor Improvements
 
+- Allow thrift reporter even if grpc hosts are not provided ([#1400](https://github.com/jaegertracing/jaeger/pull/1400), [@pavolloffay](https://github.com/pavolloffay))
+- Allow to use TLS with ES basic auth ([#1388](https://github.com/jaegertracing/jaeger/pull/1388), [@pavolloffay](https://github.com/pavolloffay))
```

Comment:
feels like new feature

Reply:
I have classified it as minor improvement :) I can move it to new features section though